litellm/llms/anthropic.py

```diff
@@ -52,7 +52,7 @@ class AnthropicConfig:
 
     def __init__(
         self,
-        max_tokens: Optional[int] = 256,  # anthropic requires a default
+        max_tokens: Optional[int] = None,  # You can pass in a value yourself or use the default value 4096
```

Comment:
if you init it as None this will raise an error. Inside the init, check if it's none and set the default